docs/alerts/monitors/settings.md

```diff
@@ -111,6 +111,13 @@ Click the **Mute** button mute the monitor. See also: [Muting Schedules](/docs/
 
 <img src={useBaseUrl('img/alerts/monitors/mute-monitor.png')} alt="mute-monitor" style={{border: '1px solid gray'}} width="500"/>
 
+### Share a monitor
+
+You can share information about a monitor with options in the [**More Actions**](#more-actions) menu:
+* **Duplicate**. Copies the monitor and gives you creator permissions on the duplicated monitor.
+* **Export**. Exports the monitor to JSON. To import a copy of the monitor to another **Monitors** folder, on the **Monitors** page select **Add > Import**.
+* **Copy Link**. Copies a link to the monitor. Provide the link to any Sumo Logic user in your organization so they can open the monitor.
```

Comment:
It looks like this duplicates info in https://help.sumologic.com/docs/alerts/monitors/settings/#more-actions. Maybe consolidate the two?

Reply:
Good idea, @kimsauce.